compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangTableLiteral.java

```diff
@@ -31,7 +32,10 @@
     public BLangExpression configurationExpr;
 
     public BLangTableLiteral() {
+    }
```

Comment:
Can we check the usage of the default constructor and remove it if possible? Someone can make a mistake if he forgot to set the type.